client/src/components/Cart.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, useState } from "react";
+import { Item, emptyItem, CartItem, emptyCartItem } from "./../interfaces";
+import {
+  Fab,
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow,
+} from "@material-ui/core";
+
+function Cart({ contents }: { contents: CartItem[] }) {
+  const getTotalPrice = () => {
+    let tot_price = 0.0;
+    for (let i = 0; i < contents.length; ++i) {
+      tot_price += contents[i].quantity * contents[i].item.price;
+    }
+    return tot_price.toFixed(2);
```

Comment:
Try to avoid using magic number, can have a const variable like currency_precision